executor/tiflashtest/tiflash_test.go

```diff
@@ -1425,3 +1425,38 @@ func TestMPPMemoryTracker(t *testing.T) {
 	require.NotNil(t, err)
 	require.True(t, strings.Contains(err.Error(), "Out Of Memory Quota!"))
 }
+
+func TestDisaggregatedTiFlashGenColumn(t *testing.T) {
```

Comment:
This test is only used to test that if a table contains virtual column, TiDB will still generate a MPP plan if the virtual column is not used?